skins/oasis/modules/templates/UserPagesHeader_BlogPost.php

```diff
@@ -1,29 +1,27 @@
 <header id="WikiaUserPagesHeader" class="WikiaUserPagesHeader WikiaBlogPostHeader">
-	<?= F::app()->renderView('CommentsLikes', 'Index', array('comments' => $comments)); ?>
-	<?php
-		if (!empty($actionMenu['action'])) {
-			echo F::app()->renderView('MenuButton', 'Index', array(
-				'action' => $actionMenu['action'],
-				'dropdown' => $actionMenu['dropdown'],
-				'image' => MenuButtonController::EDIT_ICON,
-			));
-		}
-	?>
-	<h1><?= htmlspecialchars($title) ?></h1>
+	<?= $app->renderView( 'CommentsLikes', 'Index', [ 'comments' => $comments ] ); ?>
+	<?php if ( !empty( $actionMenu['action'] ) ): ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "action" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)